### DISCUSSION TITLE
How to paste?

### DISCUSSION THREAD

**Discussion by blogdemoi** [hrkfdn/ncspot#1196] [Q&A]
Hi,

How can I paste into the search tab (I wish to copy/paste artist/song name I found on the net)?

Thank you.

**Comment by Bettehem** [comment]
You can press ctrl+v in your Library view to paste a link in.
See this page for all the keyboard shortcuts and how to customize them etc:
https://github.com/hrkfdn/ncspot/blob/main/doc/users.md#key-bindings

**Reply by blogdemoi** [reply]
Thank you. However I wish to paste text in the search tab F2. Mainly because I sometimes find interesting artists from other countries, e.g China, and can't write Chinese characters, so I need to copy/paste them, to find the artist/song in Spotify.

**Reply by Bettehem** [reply]
Then that depends on the terminal you use. For example with Xfce-terminal and Alacritty, you can use Ctrl+shift+c and ctrl+shift+v to copy/paste text, but the keybindings can be different depending on which terminal you use. Iirc gnome-terminal uses ctrl+alt+c and ctrl+alt+v

**Reply by blogdemoi** [reply]
When I try to paste, ncspot gives me an error message:

> The command "insert" is unsupported in this view

So I guess it is impossible to paste in the F2/search view/tab

**Reply by Bettehem** [reply]
> So I guess it is impossible to paste in the F2/search view/tab

No it's not. Don't use ctrl+v in the search view, but as I said earlier, you need to use the keybinding for pasting which is builtin to your terminal. So something like ctrl+shift+v or ctrl+alt+v (but that depends on the terminal you use) instead of just ctrl+v. Or if your terminal supports it, you could just right click -> paste
